caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,24 @@
+class Cipher
+  def caesar_cipher(text, usernum)
```

Comment:
@zzokki81 this is basically a function wrapped in a class. Ruby is an object-oriented programming language. Whenever you are writing Ruby code you should try to be as object-oriented as possible. Even if this particular assignment might not warrant a complex object the point of these exercises is to practice OOP.

Imagine this is a real-world object - an encryption machine. You would use it by setting the encryption parameters (i.e. rot key), then type on it (i.e. pass characters to it), and then you'd expect somewhere to see the encrypted message.

So in programming terms I would like to be able to do something like this in the main program using your class:

```
rot13 = Cipher.new(13)
rot2 = Cipher.new(2)

messages = ["xzy", "abc", ... ... ... ]

messages.each do |message|
  puts "Original: #{message}"
  puts "ROT13: #{rot13.ecrypt(message)}"
  puts "ROT2: #{rot2.ecrypt(message)}"
end
```

Please modify your class so that something like this is possible.

Finally, use more descriptive variable names than `text` or `usernum`. Variable names should reflect your intent and they should communicate that to other people reading the code. Better names would be `message`, `unencrypted_message`, ``unencrypted_text`, `raw_message`, `rot_number`, `rot_encryption`, `rot_key`, or similar (these were just some examples)